gems/bess/lib/houdini/nonprofit_creation.rb

```diff
@@ -0,0 +1,31 @@
+class Houdini::NonprofitCreation
```

Comment:
I wish we could test the CLI directly but I suppose that wouldn't be the easiest thing to do. Moving it here is an acceptable solution.